app/src/main/java/io/homeassistant/companion/android/util/NotificationActionContentHandler.kt

```diff
@@ -25,6 +25,7 @@ object NotificationActionContentHandler {
             } ?: WebViewActivity.newInstance(context)
 
             intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK
+            intent.flags = Intent.FLAG_ACTIVITY_MULTIPLE_TASK
```

Comment:
Doesn't the second line just override the first?

Reply:
It was to my understanding we needed both flags because it says its always paired with the original flag:

https://developer.android.com/reference/android/content/Intent#FLAG_ACTIVITY_MULTIPLE_TASK

Reply:
The real purpose I believe for the fix for #1472 is this comment

`Used in conjunction with FLAG_ACTIVITY_NEW_TASK to disable the behavior of bringing an existing task to the foreground.`

Reply:
oh wait, I see what you mean 🤦‍♂️ wrong syntax lol